src/main/java/net/minecraftforge/fml/client/GuiSlotModList.java

```diff
@@ -86,20 +93,24 @@ protected void drawSlot(int idx, int right, int top, int height, Tessellator tes
         if (Loader.instance().getModState(mc) == ModState.DISABLED)
         {
             font.drawString(font.trimStringToWidth(name,       listWidth - 10), this.left + 3 , top +  2, 0xFF2222);
-            font.drawString(font.trimStringToWidth(version,    listWidth - 10), this.left + 3 , top + 12, 0xFF2222);
+            font.drawString(font.trimStringToWidth(version,    listWidth - (10 + 8 + 17)), this.left + 3 , top + 12, 0xFF2222);
             font.drawString(font.trimStringToWidth("DISABLED", listWidth - 10), this.left + 3 , top + 22, 0xFF2222);
```

Comment:
10 + 8 + some_numer, as you advised me. The 17 is exactly the number needed to trim the string at the right place.

> Am 22.03.2016 um 02:06 schrieb LexManos <notifications@github.com>:
> 
> In src/main/java/net/minecraftforge/fml/client/GuiSlotModList.java:
> 
> > @@ -86,20 +93,24 @@ protected void drawSlot(int idx, int right, int top, int height, Tessellator tes
> >          if (Loader.instance().getModState(mc) == ModState.DISABLED)
> >          {
> >              font.drawString(font.trimStringToWidth(name,       listWidth - 10), this.left + 3 , top +  2, 0xFF2222);
> > -            font.drawString(font.trimStringToWidth(version,    listWidth - 10), this.left + 3 , top + 12, 0xFF2222);
> > +            font.drawString(font.trimStringToWidth(version,    listWidth - (10 + 8 + 17)), this.left + 3 , top + 12, 0xFF2222);
> >              font.drawString(font.trimStringToWidth("DISABLED", listWidth - 10), this.left + 3 , top + 22, 0xFF2222);
> Where do these values come from?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>